src/dotnet/Mgmt.CI.BuildTools/NugetToolsPackage/CI.Tools.Package/build/targets/utility.targets

```diff
@@ -26,11 +26,10 @@
   </Target>
 
   <!--Condition=" '$(PullRequestNumber)' != '' AND '$(RepoHtmlUrl)' != '' " -->
-  <Target Name="FindRPScopeFromPR" Condition=" '$(PullRequestNumber)' != '' AND '$(RepoHtmlUrl)' != '' ">
+  <Target Name="FindRPScopeFromPR" Condition=" '$(PullRequestNumber)' != '' AND '$(RepoHtmlUrl)' != '' AND '$(DetectRPScope)' == 'true' ">
```

Comment:
Where is DetectRPScope supposed to get set to true?

Reply:
Good catch, I wonder why I added that, will remove it.